client/src/ingredients/prose.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+export function Prose({ section }) {
+  return <div dangerouslySetInnerHTML={{ __html: section.content }} />;
+}
+
+export function ProseWithHeading({ id, section }) {
+  return (
+    <>
+      <h2 id={id}>{section.title}</h2>
+      <div dangerouslySetInnerHTML={{ __html: section.content }} />

```

Comment:
```suggestion
      <Prose section={section} />
```

Reply:
I know you just moved existing code. Probably written by me :)
But I noticed it now and it's a chance to get it right.